runtime/src/bank.rs

```diff
@@ -6381,22 +6380,7 @@ impl Bank {
         ancestors: &Ancestors,
         pubkey: &Pubkey,
     ) -> Option<(AccountSharedData, Slot)> {
-        match self.rc.accounts.load_with_fixed_root(ancestors, pubkey) {
-            Some((mut account, storage_slot)) => {
-                ExpectedRentCollection::maybe_update_rent_epoch_on_load(
```

Comment:
the call to `maybe_update_rent_epoch_on_load` should have been skipped if the feature was activated.